Source/AssetRipper.Processing/AnimationClips/AnimationClipConverter.cs

```diff
@@ -53,111 +53,123 @@ private void ProcessInner()
 			if (m_clip.Has_MuscleClip_C74() && m_clip.Has_ClipBindingConstant_C74())
```

Comment:
This can be condensed into:
```cs
if (m_clip.Has_ClipBindingConstant_C74())
```
because `Has_MuscleClip_C74()` is always true when `Has_ClipBindingConstant_C74()` is true.

Reply:
implemented